src/OrbitGl/include/OrbitGl/CallTreeView.h

```diff
@@ -106,59 +104,60 @@ class CallTreeNode {
   }
 
  private:
-  // absl::node_hash_map instead of absl::flat_hash_map as pointer stability is
-  // needed for the CallTreeNode::parent_ field.
-  absl::flat_hash_map<uint32_t, std::unique_ptr<CallTreeThread>> thread_children_;
-  absl::flat_hash_map<uint64_t, std::unique_ptr<CallTreeFunction>> function_children_;
+  absl::flat_hash_map<uint32_t, std::unique_ptr<CallTreeThread>> thread_children_{};
+  absl::flat_hash_map<uint64_t, std::unique_ptr<CallTreeFunction>> function_children_{};
   absl::flat_hash_map<orbit_client_data::CallstackType, std::unique_ptr<CallTreeUnwindErrorType>>
-      unwind_error_type_children_;
+      unwind_error_type_children_{};
   // std::shared_ptr instead of std::unique_ptr because absl::node_hash_map
   // needs the copy constructor (even for try_emplace).
   std::shared_ptr<CallTreeUnwindErrors> unwind_errors_child_;
```

Comment:
The comment is outdated and should be updated or removed. Verify if `std::shared_ptr` is still needed.

Reply:
Done and switched to unique_ptr